File: verify_login.js
const API_URL = 'http://localhost:5000/api';

async function testLogin(email, password) {
    console.log(`Testing login for: ${email} ...`);
    try {
        const response = await fetch(`${API_URL}/auth/login`, {
            method: 'POST',
            headers: { 'Content-Type': 'application/json' },
            body: JSON.stringify({ email, password })
        });
        const data = await response.json();
        console.log('Status:', response.status);
        if (data.error) {
            console.log('Error Message:', data.error);
        } else {
            console.log('Success:', data);
        }
    } catch (error) {
        console.log('Error:', error.message);
    }
    console.log('---');
}

async function runTests() {
    // 1. Non-existent email
    await testLogin('nonexistent' + Date.now() + '@example.com', 'somepassword');

    // 2. Wrong password (assuming 'neo' exists)
    await testLogin('neo', 'wrongpassword' + Date.now());
}

runTests();

File: verify_login.py
import urllib.request
import urllib.error
import json

def test(email, password):
    url = 'http://localhost:5000/api/auth/login'
    data = json.dumps({'email': email, 'password': password}).encode('utf-8')
    req = urllib.request.Request(url, data=data, method='POST')
    req.add_header('Content-Type', 'application/json')
    try:
        with urllib.request.urlopen(req) as f:
            print(f"Status: {f.status}")
            print(f"Response: {f.read().decode('utf-8')}")
    except urllib.error.HTTPError as e:
        print(f"Status: {e.code}")
        print(f"Error Response: {e.read().decode('utf-8')}")
    except Exception as e:
        print(f"Error: {str(e)}")

print("--- Test 1: Non-existent ---")
test('nonexistent@example.com', 'pass')
print("\n--- Test 2: Wrong password ---")
test('neo', 'wrongpass')
